dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/DefaultTrackerImportService.java

```diff
@@ -170,8 +170,12 @@ public TrackerImportParams getParamsFromMap( Map<String, List<String>> parameter
             ValidationMode.FULL ) );
         params.setImportMode( getEnumWithDefault( TrackerBundleMode.class, parameters, "importMode", TrackerBundleMode.COMMIT ) );
         params.setIdentifiers( getTrackerIdentifiers( parameters ) );
+
+        // TODO: Why is default CREATE_AND_UPDATE here and CREATE on TrackerImportStrategy?
```

Comment:
It's a very common scenario to have a mix of new and updates when sending to `/api/tracker` so this mode allows it to internally sort (if the trackedEntityInstance already exists) into updates and creates. We default to this on all our data endpoints.